src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -5,10 +5,13 @@
     <Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
     <AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-    <VersionPrefix>1.1.2</VersionPrefix>
+    <VersionPrefix>1.1.3</VersionPrefix>
     <PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
@anusapan you do not need this property anymore, so please remove it. If you get in the upstream changes, this will no longer be needed.

Reply:
reverted